api-cli.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+if [ "$#" -ne 2 ]
+then
+    echo "Usage: dump|validate|apply configuration.config"
+    exit
+fi
+case "$1" in
+    "dump")
+        curl -X GET localhost:8001/api/config --user admin:admin -o "$2"
```

Comment:
we should make localhost:8081 a parameter (with default localhost:8081)
we should also have a parameter for "username" and "password"

I think that 'dump' by default should write to stdout (without '-o')
we can have a parameter for the destination file as well, with default to stdout

it will be very easy to use stdout 

`api-cli.sh dump --user myuser --password  > myfile.conf`